awx/ui/client/features/applications/index.js

```diff
@@ -92,6 +92,14 @@ function ApplicationsRun ($stateExtender, strings) {
             }
         },
         searchPrefix: 'application',
+        params: {
+            application_search: {
+                value: {
+                    page_size: 10,
+                    order_by: 'name'
```

Comment:
Setting the page size to 10 instead of 20 was missed when I added the sort dropdown to the applications list. Per @trahman73, if the list is always "expanded" then show 10 items.